keyboards/handwired/jotpad16/jotpad16.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "quantum.h"
+
+// Used to create a keymap using only KC_ prefixed keys
+#define LAYOUT_jotpad16( \
```

Comment:
Needs to be `LAYOUT_ortho_4x4` so that `LAYOUTS = ortho_4x4` in `rules.mk` will work.

```suggestion
#define LAYOUT_ortho_4x4( \
```